Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/AutoIngestManager.java

```diff
@@ -530,7 +546,49 @@ void resume() {
     }
 
     /**
+     * Removes the priority (set to zero) of all pending ingest jobs for a
+     * specified case.
+     *
+     * @param caseName The name of the case to be deprioritized.
+     *
+     * @throws AutoIngestManagerException If there is an error removing the
+     *                                    priority of the jobs for the case.
      */
+    void deprioritizeCase(final String caseName) throws AutoIngestManagerException {
+        if (state != State.RUNNING) {
+            return;
+        }
+
+        List<AutoIngestJob> prioritizedJobs = new ArrayList<>();
```

Comment:
I feel like naming this (and the other similar objects in the other methods) something like "jobsToDeprioritize" would make the code clearer.